kong/runloop/handler.lua

```diff
@@ -165,9 +165,16 @@ return {
       local worker_events  = singletons.worker_events
       local cluster_events = singletons.cluster_events
 
+      -- database topology refresher
 
-      -- events dispatcher
+      worker_events.register(function()
+        local ok, err = dao.db:refresh()
+        if not ok then
+          ngx.log(ERR, "[events] failed to schedule DB refresh: ", err)
```

Comment:
If the cluster topology fails to refresh, it warrants a higher logging level. That is because more than one transaction/request might be affected by this error. In fact, this error may trigger cascading failures in the long run. A logging level set to `ngx.CRIT` seems more appropriate here.

Also, the error message seems wrong: it isn't the scheduling that failed in this case, but the refresh itself. And the other way around: the message currently showed in the Admin API failure branch seems to be wrong; in this case, it is indeed the _scheduling_ that failed, and not the refresh itself.